datafusion/sqllogictest/test_files/interval.slt

```diff
@@ -44,7 +44,7 @@ Interval(MonthDayNano) Interval(MonthDayNano)
 query ?
 select interval '5' years
 ----
-5.000000000 secs
+60 mons
```

Comment:
this certainly seems much more correct to me -- I also removed the now out of date comment.